### ISSUE TITLE
Add a new command to copy Notepad++/System info into clipboard for bug report

### ISSUE THREAD

**Issue by donho** [notepad-plus-plus/notepad-plus-plus#694]
A new command "Info to Clipboard" could be added in "?" Menu or in About box as a button.

The expected info into clipboard are:
```
Notepad++ version: 6.8.1
Build time: Aug 3 2015 - 23:43:31
Path: C:\Programs\Notepad++\notepad++.exe
Admin mode: No
Local Conf mode: No
Plugins: NppFTP, Mime tool, Converter, NppExport
OS: Microsoft Windows 7 Professionnel 64 bit
Version: 6.1.7601 Service Pack 1 Build 7601
Compiler: VS2013
```

This component could be done as plugin. In this case, only a command item will be added in Plugin Menu.

**Comment by cmeriaux** [comment]
You should add the git sha1sum which is the most reliable information.

**Comment by donho** [comment]
Include sha1sum is a good idea but it'll need (much) more effort to implement it.

**Comment by milipili** [comment]
Actually, it would be nice to be in markdown format for an easy copy/paste, and a more compact format. It would help to have clean bug report on github. May something like that :

> **Notepad++ version: 6.8.1** _(build vs2013 - Aug 3 2015 - 23:43:31)_
> os: Microsoft Windows 7 Professionnel 64 bit _(6.1.7601 Service Pack 1 Build 7601)_
> path: `C:\Programs\Notepad++\notepad++.exe`
> admin mode: no
> local conf mode: no
> plugins: **`NppFTP`**, **`Mime tool`**, **`Converter`**, **`NppExport`**

Or if there is no plugin:

> **Notepad++ version: 6.8.1** _(build vs2013 - Aug 3 2015 - 23:43:31)_
> os: Microsoft Windows 7 Professionnel 64 bit _(6.1.7601 Service Pack 1 Build 7601)_
> path: `C:\Programs\Notepad++\notepad++.exe`
> admin mode: no
> local conf mode: no
> plugins: _none_

**Comment by GitMensch** [comment]
This "plugin" is for PyQT apps and written in Python, therefore not usable here at all - but as it has everything a good "report issue" should have I still wanted to link it as reference:

https://github.com/ColinDuquesnoy/QCrash/

**Comment by dail8859** [comment]
Closed by 773397d6eab4bd59ea7104796936d9d52456c228
